virt-v2v/cold/BUILD.bazel

```diff
@@ -92,7 +92,13 @@ container_image(
 
 go_library(
     name = "cold_lib",
-    srcs = ["entrypoint.go"],
+    srcs = [
+        "customize-image.go",
```

Comment:
I suppose it will be needed also for warm migrations, right?

Reply:
@ahadas @mnecas can we do the fix for the warm migration in a separate PR ? does MTV-1299 require a fix for the warm case too ?

Reply:
Yeah we can split it but Arik had good comment that we need to think about other migration types so we need to move out these scripts out of cold and make some utils which can be shared in both of them.

Reply:
I'll make it in separate MR